faqs/csv_parser/csv_parser.py

```diff
@@ -0,0 +1,85 @@
+import csv
+
+
+class CsvContentParser:
```

Comment:
Classes should inherit from `object` (e.g. `CsvContentParser(object)`). Classes that inherit from `object` are "new-style classes" and support many new features (e.g. have richer attribute look-up, are proper types, better multiple inheritance). Classes that don't inherit from `object` are "old-style classes" and are generally less cool.

Aside: This isn't necessary in Python 3 since all classes implicitly inherit from `object`, but presumably we'd still like to support Python 2 for a bit.